tests/Unit/SimplePieTest.php

```diff
@@ -206,7 +206,7 @@ public function testItemWithEmptyContent(): void
         $content = 'item description';
         $feed = $this->createFeedWithTemplate($data, $content);
         $item = $feed->get_item();
-        $this->assertSame($content, $item->get_content());
+        $this->assertSame($content, $item ? $item->get_content() : null);
```

Comment:
Probably cleaner to do assertInstanceOf Item like we do elsewhere.

Reply:
This test seems to be specifically checking that the content is the same as the control.  Would you like to use `assertInstanceOf` instead, or in addition to?

Reply:
In addition to. This would ensure that `get_item()` actually returns `Item`, and not `null`, so we would be sure we can call `get_content()` on it.

Reply:
Got it, done ✅